crowdsec-docs/unversioned/user_guides/alert_context.md

```diff
@@ -4,41 +4,30 @@ title: Contextualize Alerts
 sidebar_position: 10
 ---
 
-# Alert Contextualization
+## Introduction
 
-While CrowdSec doesn't store any logs after processing it, it can be useful to get some context about why an alert has been triggered (source machine, targeted FQDN ...).
+CrowdSec doesn't store raw log information once it has finished processing them. However, you can send additional data within an alert with a feature we call **alert context**:
 
-The context configuration is part of the CrowdSec Hub and each collection should come with a context configuration file.
+![Alert Context Example](/img/alert_context.png)
 
-The only thing to do if you want to see the context in the console is to enable the `context` option in your CrowdSec Local API [see below](#send-alert-context-to-crowdsec-console).
+While some collections already include an alert context configuration to join the relevant information to an alert, you have to explicitly enable it:
```

Comment:
Whilst some collections already include a context configuration, you must enable the `context` option within your security engine console configuration.

This can be achieved by running:
...